src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -320,8 +323,8 @@ public RestoreCommand(IApplicationEnvironment env)
             var lockFile = await ReadLockFile(projectLockFilePath);
 
             var useLockFile = false;
-            if (Lock == false && 
-                Unlock == false && 
+            if (Lock == false &&
```

Comment:
There's a simpler way to evaluate `bool`s :smile: (The code was already there, I know.)